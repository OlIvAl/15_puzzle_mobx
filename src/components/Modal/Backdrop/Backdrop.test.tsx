import * as React from 'react';
import Backdrop from '.';
import renderer from 'react-test-renderer';

describe('Backdrop', () => {
  it('он отображается', () => {
    const tree = renderer.create(<Backdrop />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});