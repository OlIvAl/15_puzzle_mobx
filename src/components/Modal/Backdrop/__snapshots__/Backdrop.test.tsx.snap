// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop он отображается 1`] = `
.c0 {
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 100vh;
  background-color: rgba(0,0,0 ,0.5);
  z-index: -1;
}

<div
  className="c0"
/>
`;
